assets/license-header-rustfmt-template

```diff
@@ -0,0 +1,4 @@
+// <project> is Copyright © {[-,0-9]+} {.+}
+//
+// This file is part of <project>, distributed under the GPL v3 with Radicle Linking Exception.
+// For full terms see the included LICENSE file.
```

Comment:
https://github.com/radicle-dev/radicle-decisions/pull/8 proposes a universal solution to the same problem. If we decied to merge it, will the rustfmt template still be useful?

Reply:
I wouldn't insist that this decision act has to prescribe exactly what tooling has to be used, it's more like a copy&pasta source. I'll definitely leave the rustfmt check in place, because it's more likely contributors would run rustfmt before submitting, than to set up any hooks or find the ci scripts.

Reply:
Good point, it's a set of tools